Wikipedia/Code/AppearanceSettingsViewController.swift

```diff
@@ -72,8 +72,14 @@ open class AppearanceSettingsViewController: UIViewController, UITableViewDataSo
     
     func sectionsForAppearanceSettings() -> [AppearanceSettingsSection] {
         
+        func checkmarkItemFor(theme: Theme) -> (AppearanceSettingsCheckmarkItem) {
```

Comment:
Not a blocker for merge, but this method signature would also be acceptable:

> func checkmarkItem(for theme: Theme)

Reply:
:+1: good catch, that's more swifty ;)